artifactory_test.go

```diff
@@ -3605,6 +3607,32 @@ func TestSummaryReport(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+func TestUploadDetailedSummary(t *testing.T) {
+	initArtifactoryTest(t)
+	uploadCmd := generic.NewUploadCommand()
+	fileSpec := spec.NewBuilder().Pattern("testdata/a/a*.in").Target(tests.RtRepo1).BuildSpec()
+	uploadCmd.SetUploadConfiguration(createUploadConfiguration()).SetSpec(fileSpec).SetServerDetails(serverDetails).SetDetailedSummary(true)
+	commands.Exec(uploadCmd)
+	result := uploadCmd.Result()
+	reader := result.Reader()
```

Comment:
Check errors + close reader:
```suggestion
	reader := result.Reader()
	assert.NoError(t, reader.GetError())
	defer reader.Close()
	
```